wp-content/themes/workingnyc/assets/js/global-9c3ebc0.js

```diff
@@ -1274,6 +1274,19 @@
     googleTranslateElementInit();
   }
 
+ /** Toggle the language dropdown carrot */
+
+  let toggleElement = document.querySelector('[data-js="dialog"]');
```

Comment:
It's not working locally for me. I think the solution is to use a more specific attribute: `data-js="dialog"` is used in several different elements, not just on the language dropdown, so maybe add something like `data-js-dialog="language"` here and in the `<button>` tag of each of the twig files.

Reply:
Thanks for the solution, it's good to add new data attribute to select the right element.